json-endpoints/class.wpcom-json-api-site-settings-v1-3-endpoint.php

```diff
@@ -0,0 +1,159 @@
+<?php
+/**
+ * WARNING: This file is distributed verbatim in Jetpack.
+ * There should be nothing WordPress.com specific in this file.
+ *
+ * @hide-in-jetpack
+ * @autounit api-v1 site-settings
+ */
+
+new WPCOM_JSON_API_Site_Settings_V1_3_Endpoint( array(
+	'description' => 'Get detailed settings information about a site.',
+	'group'       => '__do_not_document',
```

Comment:
Does it make sense for publicly released source code of endpoints to be flagged `__do_not_document` ?

Reply:
(I don't know enough about how the `group` field works to know the answer to this -- maybe it totes does)

Reply:
I don't know - will investigate further why we treat the settings endpoints this way